.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release & Publish to NPM
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+env:
+  PNPM_CACHE_FOLDER: .pnpm-store
+  HUSKY: 0 # Bypass husky commit hook for CI
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+      - name: Initialize Git User
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "alvaro.saburido@gmail.com"
```

Comment:
My suggestion would be to pack the user.email also in an env variable or secret, that way if in the future you want to change it or maybe us an organisation email (github(at).tresjs.org) or something else you can easily change it without having to update the code.